csrc/device_lower/lower2device.cpp

```diff
@@ -589,6 +584,11 @@ void GpuLower::analysis(Fusion* fusion) {
     tensor_indexer_ = std::make_unique<TensorIndexer>(*id_model_);
   }
 
+  // Detects all exprssions that don't need predicates. Depends on
```

Comment:
Please mention this now also depends on tensor_indexer_.

Reply:
I followed your suggestions so I no longer depend on tensor_indexer_. So I reverted the changes to this file.